Readme.md

```diff
@@ -4,8 +4,10 @@
 
 SaltyRTC is an **end-to-end encrypted signalling protocol**. It offers
```

Comment:
Meh, we should change this to:

> **end-to-end encrypted transport and signalling protocol**

But I think it's too late for that. Moving that to v2.

Reply:
Why? It's just the README. And it's true for v1 too.

Reply:
Because it requires a lot of changes. I mean, sure, go for it. :grin: